### DISCUSSION TITLE
[Feature] docker-compose.yaml - volume variable for immich-redis volume

### DISCUSSION THREAD

**Discussion by Parlendir** [immich-app/immich#3217] [Feature Request]
### The feature

Hello

The docker-compose.yaml file proposed with each release does not contain any volume variable for the immich-redis container.

De facto, docker creates a temporary volume in its working directory.

REDIS data is considered dispensable during a release upgrade or redeployment, but such a variable would allow the user to choose, or not, the directory for this data, and at the very least its naming, rather than leaving docker the task of placing and naming this volume.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
For simplicity, the standard compose setup is deliberately kept as minimal as we can. Users are free to edit it as they please, of course.

**Reply by vogler** [reply]
Just give it a name - leaving anonymous volumes around is messy:
<img width="551" alt="image" src="https://github.com/user-attachments/assets/404f94d3-4b8c-4eb7-8757-766bc69af4c8">
